installer/dev/WindowsAppRuntimeInstall.vcxproj

```diff
@@ -76,7 +76,7 @@
       <PrecompiledHeaderFile>pch.h</PrecompiledHeaderFile>
       <PrecompiledHeaderOutputFile>$(IntDir)pch.pch</PrecompiledHeaderOutputFile>
       <PreprocessorDefinitions>_CONSOLE;WIN32_LEAN_AND_MEAN;WINRT_LEAN_AND_MEAN;%(PreprocessorDefinitions)</PreprocessorDefinitions>
-      <AdditionalOptions>%(AdditionalOptions) /bigobj</AdditionalOptions>
+      <AdditionalOptions>%(AdditionalOptions) /bigobj /Qspectre</AdditionalOptions>
```

Comment:
Not using the root's Directory.Build.props is bad (for this and other reasons), no? See other comment

P.S. MRTCore has the same issue?

Reply:
Separate bugs filed for both installer and MRT to use the root's props files.

Reply:
Also nuget.config. There's 4 copies in the tree. We should only be using the 1 in the root.